src/query/procedure/py_module.cpp

```diff
@@ -2052,6 +2052,86 @@ PyObject *PyPathMakeWithStart(PyTypeObject *type, PyObject *vertex) {
   return py_path;
 }
 
+// clang-format off
+struct PyLogger {
+  PyObject_HEAD
+};
+// clang-format on
+
+void PyLoggerAddLog(const enum mgp_log_level level, const char *out) { mgp_log(level, out); }
+
+PyObject *PyLoggerAddInfoLog(PyLogger *self, PyObject *args) {
+  const char *out = nullptr;
+  if (!PyArg_ParseTuple(args, "s", &out)) return nullptr;
```

Comment:
```suggestion
  if (!PyArg_ParseTuple(args, "s", &out)) {
    return nullptr;
  }
```

I am not 100% sure about formatting, so directly applying this commit might not be correct.

The same is true for the rest of the functions.